### ISSUE TITLE
Closing and reopening CodeRoad sends me to the first set of instructions

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/CodeAlly-CodeRoad-freeCodeCamp#19]
When I close CodeRoad and start it back up. I get the instructions for the first lesson...
[Here's a gif](https://files.gitter.im/5c924148d73408ce4fbb3fa5/9kIH/Apr-01-2021-09-10-15.gif)

All the files are there for the lesson I left off from, so that part is in the right spot.

I don't believe this is the behavior locally - I think continuing a tutorial there will work.

**Comment by AdamZaczek** [comment]
I believe it's an issue on the CodeAlly part. We'll take a look.

**Comment by moT01** [comment]
Side note: This doesn't happen locally

**Comment by AdamZaczek** [comment]
We made a fix and we can not reproduce it anymore.
Do you still experience the issue?

**Comment by moT01** [comment]
I'm wondering if it's cause when we load tutorials on CodeAlly, we skip one step at the beginning. If I run locally, I see a "start new tutorial" and "continue tutorial" options. When running on CodeAlly, it just auto-loads. Does that sound like it might be a cause @ShMcK?

**Comment by ShMcK** [comment]
@moT01 like you mentioned - its because of the logic for skipping a step at the beginning.

```
if (TUTORIAL_URL) {
        // if a tutorial URL is added, launch tutorial on startup
        ...
}
```

CodeAlly launches with a TUTORIAL_URL of which tutorial to use, which you probably don't have set locally.

I'll put in a fix to check to just auto-continue for users.

**Comment by ShMcK** [comment]
@AdamZaczek let me know if this works with your testing as of CodeRoad v0.16+

**Comment by ShMcK** [comment]
Released in CodeRoad v0.17 on the ovsx marketplace
